libraries/src/Table/Content.php

```diff
@@ -247,17 +255,17 @@ public function check()
 		// Set publish_up to null date if not set
 		if (!$this->publish_up)
 		{
-			$this->publish_up = $this->_db->getNullDate();
+			$this->publish_up = null;
 		}
 
 		// Set publish_down to null date if not set
 		if (!$this->publish_down)
 		{
-			$this->publish_down = $this->_db->getNullDate();
+			$this->publish_down = null;
 		}
 
 		// Check the publish down date is not earlier than publish up.
-		if ($this->publish_down < $this->publish_up && $this->publish_down > $this->_db->getNullDate())
+		if (!is_null($this->publish_up) && !is_null($this->publish_up) && $this->publish_up > $this->publish_down)
```

Comment:
I think we use === NULL not is_null in code style or?

```suggestion
		if ($this->publish_up !== NULL && $this->publish_up !== NULL && $this->publish_up > $this->publish_down)
```

Reply:
@HLeithner We use both it seems, but is_null() we use by far more often.

Reply:
iirc @mbabker changed it in the frame work from is_null to !== NULL I'm and sure if the coding style says anything about this. At least we should use the same in all places.

Reply:
`is_null()` is slower. Also `null` must be in lowercase.

Reply:
Our codestyle has no rules regarding that. If you want us to refactor to that, then I would ask to make a separate issue for that. You might want to bring that up in the ATT. 😉